File: crates/ctxctl/tests/fixtures/sample.go
// Package fixture is used by ctxctl integration tests.
package fixture

// Point is a 2D point.
type Point struct {
	X float64
	Y float64
}

// Norm returns the distance from the origin.
func (p *Point) Norm() float64 {
	return p.X*p.X + p.Y*p.Y
}

// Add sums two integers.
func Add(a, b int) int {
	return a + b
}

const MaxRetries = 3

File: crates/ctxctl/tests/fixtures/sample.js
// Fixture module used by ctxctl integration tests (javascript).

import express from "express";
import { helper } from "./helpers";
const os = require("os");

/** A user entity. */
class User {
  /** Say hello. */
  greet() {
    return "hi";
  }
}

export function formatName(user) {
  return user.name.trim();
}

const MAX_RETRIES = 3;
